advanced/accessors/01_accessor_examples.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30. Parameters
Line #31. ----------           
Line #32. req_dim : list of str
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>